Added Pretrained Model Example Submission

aadi07 opened the pull request:
Added an example extending the base one that includes a .load method that can load pretrained model from an existing file. Confirm if format is ok, should be generic enough to support arbitrary libraries. We could also put up a zip file to indicate that they must upload their pretrained model as well in a parallel file to model.py.

Commit:
added load model method

katyagovorkova commented:
Thanks for the changes @aadi07! One comment: I think we don't need the fit method since we are not going to retrain the submitted models. So specifying only the predict and load methods should be enough.

Commit:
added full pretrained example

aadi07 commented:
Got rid of the fit method, added a zipped pretrained example (with the caveats I sent on slack), and modified ingestion.py to load the pretrained model. Wasn't able to test it without access to the training data, but the model doesn't throw any errors when I load it (just not sure if it's what we expect it to be).

Commit:
updated ingestion and scoring programs for codabench

aadi07 commented:
Updated the ingestion and scoring programs and formatted them adjacent to metadata.yaml for convenient uploading to codabench
